apps/docs/src/.vitepress/env.ts

```diff
@@ -1,5 +1,12 @@
-export const vitepressEnv = {
-  storybookHost:
-    import.meta.env.VITE_STORYBOOK_HOST ||
-    "https://onyx-storybook-prod.apps.01.cf.eu01.stackit.cloud",
-} as const;
+/**
+ * Gets the Storybook host from the VITE_STORYBOOK_HOST env variable.
+ * If env variable is not set, a fallback value will be used.
+ */
+export const getStorybookHost = (): string => {
+  let envValue: string | undefined;
+
+  if (import.meta.env) envValue = import.meta.env?.VITE_STORYBOOK_HOST;
+  else if (process) envValue = process.env.VITE_STORYBOOK_HOST;
+
+  return envValue || "https://onyx-storybook-prod.apps.01.cf.eu01.stackit.cloud";
```

Comment:
<img width="781" alt="image" src="https://github.com/SchwarzIT/onyx/assets/151019977/bd25fe31-aed9-4a94-9827-3bcee6035d64">

my env.local has it set but the link in the top bar still directs me to the prod link...

<img width="166" alt="image" src="https://github.com/SchwarzIT/onyx/assets/151019977/10eec444-62c4-4d56-b4cb-f1ba8b4685ea">
<img width="450" alt="image" src="https://github.com/SchwarzIT/onyx/assets/151019977/9bb65781-9d18-4a26-8646-bb51f17c1af5">

Reply:
Hhm...it seems like VitePress parses the config before the env variables are read...

Reply:
Fixed